### DISCUSSION TITLE
Sitemap schema

### DISCUSSION THREAD

**Discussion by RickCogley** [lumeland/lume#553] [Ideas]
Regarding the schema lines in the sitemap generated by lume sitemap plugin: 

https://github.com/lumeland/lume/blob/a5ff1c814e1d1f4d205c0ea995c5d3aad507f8a9/plugins/sitemap.ts#L77

... which give: 

```
<?xml version="1.0" encoding="UTF-8"?>
<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml">
  <url>
    <loc>
...
```

I was wondering did you consider adding other schemas: 

```
<?xml version="1.0" encoding="UTF-8"?>
<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:news="http://www.google.com/schemas/sitemap-news/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml" xmlns:mobile="http://www.google.com/schemas/sitemap-mobile/1.0" xmlns:image="http://www.google.com/schemas/sitemap-image/1.1" xmlns:video="http://www.google.com/schemas/sitemap-video/1.1">
  <url>
    <loc>
...
```

... so that the sitemap.xml displays better in browsers? I know sitemap.org's examples are simple like lume's default, and that it validates, but I just wanted to know.

**Comment by oscarotero** [comment]
I didn't know these schemas, but they seem like are for specific pages (images, videos, news, etc).
